src/libraries/System.Net.Sockets/tests/FunctionalTests/Connect.cs

```diff
@@ -84,6 +84,7 @@ public async Task Connect_MultipleIPAddresses_Success(IPAddress listenAt)
         }
 
         [Fact]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/55053", TestPlatforms.AnyUnix)]
```

Comment:
Did these ever fail on macOS, or only on Linux?

Reply:
Same here, only Linux. Happy to update it to something better ...